doc/devdocs/disk-usage-footprint.md

```diff
@@ -1,40 +1,71 @@
-# PowerToys disk usage footprint review
+# PowerToys disk usage footprint overview
 
-Starting with the release v0.66 of PowerToys, .NET runtime dlls are being shipped self-contained as a part of PowerToys. .NET dlls are being installed in `<PowerToysInstallDir>/dll/dotnet`. During the installation process, hard-links are being created for every module that needs .NET libraries.
+As of the v0.66 of PowerToys, core runtime dlls are being shipped self-contained as a part of PowerToys. During the installation process, hard-links are being created for every module that needs any of these shared libraries.
 
-This adds .NET to the list of dependencies we are shipping self-contained (together with WinAppSDK and VC Redistributable) in PowerToys by using the hard-links technology to share the dlls between the modules in a way that saves disk space when compared to having actual copies for each module. Our main motivation for doing this is to eventually reach a point where PowerToys can be installed without requiring UAC elevation and without installing additional dependencies like .NET.
+## Why do this work?
```

Comment:
At first read, this sounds like written by a foreigner. Do you mean (something like) : **Why do/did/should we do this work** or maybe **Why _does_ this work**?